index.html

```diff
@@ -884,7 +884,30 @@ <h2>The <code>application/credential-claims-set-1.1+json</code> Media Type</h2>
   </table>
   <div class="issue" data-number="42"></div>
 </section>
+</section>
 
+<section id="appendix">
+  <h2>Appendix</h2>
+  <section>
+    <h3>Example VC Data Model Mapping</h3>
+    <p>This is one possible unidirectional mapping between 2.0 VC-JWTs and the VC Data Model; other such mappings are possible.</p>
+    <pre class="example">
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    {
+        "iss": {"@id": "https://www.w3.org/2018/credentials#issuer"},
+        "sub": {"@id": "https://www.w3.org/2018/credentials#credentialSubject"},
+        "nbf": {"@id": "https://www.w3.org/2018/credentials#issuanceDate"}
+    }
+  ],
+  "iss": "did:example:123",
+  "sub": "did:example:456",
```

Comment:
The `sub` mapping is non-conformant with the VC Data Model specification because `credentialSubject` can't be a string: https://w3c.github.io/vc-data-model/#credential-subject as processed via this issue: https://github.com/w3c/vc-data-model/issues/762 and resolved by consensus in this PR: https://github.com/w3c/vc-data-model/pull/970 and this PR: https://github.com/w3c/vc-data-model/pull/976

Reply:
I might need to write a javascript program to do this, since JSON-LD alone does not seem to work.

Would an example javascript problem be an acceptable demonstration of a mapping?

Reply:
Just confirming, seems empty object is valid, in other words:

```
credentialSubject: {}
```

1. it is an object.
2. id is optional.

Reply:
> Would an example javascript problem be an acceptable demonstration of a mapping?

So, at some point full translation rules / algorithm will need to be provided that could be implemented in a particular language anyway (so people can run them in their digital wallets, etc. to consume the data).

Starting from a JS example and then working that into prose that could be implemented in any language seems like a fine start to me personally. The spec couldn't "bless" a particular implementation, of course. If the rules are simple enough to demonstrate their implementation in an example program, I think that would be fine -- but others may say the translation algorithm in prose alone would be better than having that example.

Reply:
The VCDM now says that `credentialSubject` must have at least one claim. IMO, if folks are using `id` in a `credentialSubject` to represent a DID for example, then this becomes a claim and we should consider updating the spec to say something about `id` becomes a claim under certain circumstances.